crates/metadata/src/specs.rs

```diff
@@ -171,7 +171,7 @@ impl<S> ContractSpecBuilder<S> {
     /// Sets the documentation of the contract specification.
     pub fn docs<D>(self, docs: D) -> Self
     where
-        D: IntoIterator<Item = &'static str>,
+        D: IntoIterator<Item = String>,
```

Comment:
Could add an extra generic param `S: ToString` and then `Item = S` to make it a non breaking api change?

Reply:
Good Idea I'll try that